testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -126,6 +127,25 @@ QUnit.module('init', {
             position: { of: '#non-exist' }
         }).dxLoadPanel('instance');
     });
+
+    QUnit.test('show warning if deprecated \'elementAttr\' option is used', function(assert) {
```

Comment:
I think it's good idea to use `"` instead of `\'` to simplify test search by `ctrl+f`

Reply:
done

Reply:
There should be `'` to be equal to the string, generated with `alias` deprecation option. I also tried to use `"` as external quotes and `'` as internal, but that way is not allowed by linter.
![image](https://user-images.githubusercontent.com/29150145/116051916-cde10400-a681-11eb-94f7-90a43b2ba365.png)

Reply:
Comment above is about quotes for `wrapperAttr`. For `elementAttr` i made `"` quotes.